sanic/app.py

```diff
@@ -4,12 +4,14 @@
 import re
 import warnings
 
-from asyncio import CancelledError, ensure_future, get_event_loop
+from asyncio import CancelledError, Protocol, ensure_future, get_event_loop
 from collections import defaultdict, deque
 from functools import partial
 from inspect import getmodulename, isawaitable, signature, stack
-from ssl import Purpose, create_default_context
+from socket import socket
+from ssl import Purpose, SSLContext, create_default_context
 from traceback import format_exc
+from typing import Any, Optional, Type, Union
```

Comment:
To "shorten" code (just) a little **and** to "encourage" developers to use more `typing` hints, I always do the following:

```python
import typing as t
...
def foo(
    bar: t.Type[object],
    baz: t.Optional[int],
):
    pass
```

Does that makes sense?